Formula/partio.rb

```diff
@@ -29,6 +19,8 @@ class Partio < Formula
   def install
     args = std_cmake_args
     args << "-DPARTIO_USE_GLVND=OFF" unless OS.mac?
+    args << "-DPARTIO_GTEST_ENABLED=ON"
+    args << "-DGTEST_LOCATION=#{Formula["googletest"].opt_prefix}"
```

Comment:
Why is this needed?

Reply:
The build scripts (currently ~ in v1.17.0 and older) fail to build when using an alternate (non-`/usr/local`) brew location without specifying the googletest location.

There's already a fix[1] merged upstream that makes googletest truly optional (rather than the build error that we'd hit in 1.17.0 and older) so it's not too big deal since v1.17.1 won't try to use googletest unless it's available.

I updated the commit message to mention these details, but I also closed this MR since v1.17.1 will make it obsolete.

[1] https://github.com/wdas/partio/commit/2200a2349be560ece792d4f174a8daa8af76c33a